PCIe 6.0 Packet Generator and Exerciser/PCIe 6.0 Packet Generator and Exerciser/PCIe 6.0 Packet Generator and Exerciser.vcxproj

```diff
@@ -13,7 +13,7 @@
   <PropertyGroup Label="Globals">
     <ProjectGuid>{325867CC-7275-42C1-BB3E-F92DAC1317BF}</ProjectGuid>
     <Keyword>QtVS_v304</Keyword>
-    <WindowsTargetPlatformVersion Condition="'$(Configuration)|$(Platform)' == 'Debug|x64'">10.0.19041.0</WindowsTargetPlatformVersion>
+    <WindowsTargetPlatformVersion Condition="'$(Configuration)|$(Platform)' == 'Debug|x64'">10.0</WindowsTargetPlatformVersion>
```

Comment:
Actually this one which made conflict earlier, so we need to take care of it when merging.

Reply:
You can resolve this conflict by changing this version, and the next branches (not commits in this branch) will be with version 10.0.19041.0.

Reply:
yes I already solved it previously so if it makes a problem we can solve it